snapshot/cli/cli.go

```diff
@@ -119,3 +127,36 @@ func (c *ingestGitCommitCommand) run(db snapshot.DB, _ *cobra.Command, _ []strin
 	fmt.Println(id)
 	return nil
 }
+
+type exportGitCommitCommand struct {
+	id string
+}
+
+func (c *exportGitCommitCommand) register() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "export_git_commit",
```

Comment:
Yes, but the dataRepo and targetRepo are the same for cli, and if the point isn't to interact with store/scoot these operations aren't very meaningful. Am I misreading something?